controlsfx/src/main/java/org/controlsfx/control/textfield/TextFields.java

```diff
@@ -28,10 +28,6 @@
 
 import impl.org.controlsfx.autocompletion.AutoCompletionTextFieldBinding;
 import impl.org.controlsfx.autocompletion.SuggestionProvider;
-

```

Comment:
Since you are modifying this file, replace the `2015` in the copyright to `2020`.

Reply:
done